compiler/src/dotty/tools/dotc/tastyreflect/TastyImpl.scala

```diff
@@ -140,9 +135,11 @@ class TastyImpl(val rootContext: Contexts.Context) extends scala.tasty.Tasty wit
 
   type Definition = tpd.Tree
 
-  object Definition extends DefinitionExtractor {
-    def unapply(x: Definition)(implicit ctx: Context): Boolean =
-      x.isInstanceOf[Trees.MemberDef[_]]
+  object IsDefinition extends IsDefinitionExtractor {
+    def unapply(tree: Tree)(implicit ctx: Context): Option[Definition] = tree match {
+      case tree: tpd.MemberDef => Some(tree)
+      case _ => None
+    }
```

Comment:
I modified `PackageDef` to `PackageDefinition`.

Reply:
That's fine, but we have `type PackageDef = PackageDefinition`. You're thinking of `type PackageClause = tpd.PackageDef`, but that's unrelated. Sadly I don't see an easy refactoring to avoid this confusion...